### PR TITLE
Type definition improvements

### LINKED ISSUE #456: Type definition improvements.
Override props in order to use custom theme.

### PR DESCRIPTION
**Pull request by kamolesh97**
closes #456

### CODE REVIEW

**Comment by kamolesh97** [comment]
@sangameshsomawar _a please test and merge.

File: index.d.ts [hunk 1]
```diff
@@ -67,10 +67,32 @@ const theme: {
...
-    ColorProps,
+    Omit<
+      BorderProps,
+      | "borderColor"
+      | "borderTopColor"
+      | "borderBottomColor"
+      | "borderLeftColor"
+      | "borderRightColor"
+    >,
+    Omit<ColorProps, "backgroundColor" | "bg" | "color">,
```
Comment (sangameshsomawar) [inline index.d.ts:78]: @kamolesh97 I guess without omitting also it will work, As we are overriding at line  81. Please check once

**Comment by kamolesh97** [comment]
@sangameshsomawar updated PR

**Review by sangameshsomawar — APPROVED** [review]
@kamolesh97 LGTM, merging this PR.

**Pull request merged**